fragment.go

```diff
@@ -1104,6 +1104,107 @@ func (f *fragment) importSetValue(columnID uint64, bitDepth uint, value int64, c
 	return changed, nil
 }
 
+// distinct returns a list of distinct values in the fragment.
+// A bitmap can be passed in to optionally filter the computed columns.
+func (f *fragment) distinct(filter *Row, bitDepth uint) ([]int64, error) {
```

Comment:
How about the performance about this function? For 10,000,000 columns, how long does it spend to get all distinct values? Thanks.

Reply:
Hey @brucenan we actually rejected this implementation in favor of another, more performant approach. That implementation is currently only available in Molecula, our commercial offering which uses a private version of Pilosa.